Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.1.0RC5-apache
+FROM php:8.0-apache
```

Comment:
why did you go back  to 8.0? This will have effect on the version used for analysis on psalm.dev I believe

Reply:
psalm.dev runs 8.0:
![image](https://user-images.githubusercontent.com/57403/153200855-e5afd80a-eae7-4d88-8b51-7fc51b18be1f.png)
The version specified in Dockerfile affects dev setups only. And I prefer it to match the production version, especially because it affects what gets installed during composer updates (for `vendor-bin/**` packages that do have their versions locked).

Reply:
oh, I didn't expect that. Good to know